packages/morph/src/styles/editor.scss

```diff
@@ -1,76 +1,82 @@
 $line-height: 1.3;
-$blockquote-border-color: #cccccc;
-$blockquote-text-color: #555555;
+$gray-light: #e5e5e5;
+$gray-medium: #cccccc;
+
+$base-size: 1rem;
+$scale-ratio: 1.2;
+
+$h6-size: $base-size;
+$h5-size: $base-size * $scale-ratio;                   
+$h4-size: $base-size * $scale-ratio * $scale-ratio;
+$h3-size: $base-size * pow($scale-ratio, 3);
+$h2-size: $base-size * pow($scale-ratio, 4);
+$h1-size: $base-size * pow($scale-ratio, 5);
```

Comment:
I was thinking about this https://type-scale.spencermortensen.com/, but there are some more magic scaling based on screen type (by Utopia https://utopia.fyi/blog/designing-with-fluid-type-scales/)